packages/headless/src/features/facets/facet-set/facet-set-selectors.ts

```diff
@@ -77,5 +87,9 @@ export const isFacetLoadingResponseSelector = (state: FacetResponseSection) => {
     return state.productListing.isLoading;
   }
 
+  if ('commerceFacets' in state) {
+    return false;
+  }
```

Comment:
I'm not too sure how we should handle this case. We don't have access to the `isLoading` state on the `CommerceFacetSection`. Should we change the state to something like this?
```typescript
export type FacetResponseSection =
  | SearchSection
  | ProductListingSection
  | (CommerceFacetSection & ProductListingV2Section);
  ```
  which would eventually become:
  ```typescript
export type FacetResponseSection =
  | SearchSection
  | ProductListingSection
  | (CommerceFacetSection & (ProductListingV2Section | CommerceSearchSection));
  ```

Reply:
Perhaps for now, yes, but this makes me reconsider the way we're structuring the commerce app state.

I'll come up with something and show it to you later today 😄